RetailCoder.VBE/Inspections/ImplicitPublicMemberInspectionResult.cs

```diff
@@ -45,14 +45,13 @@ public override void Fix()
             var oldContent = Context.GetText();
```

Comment:
If the context spans the entire body, this fix is now rewriting the entire thing and deleting all comments in the process, when really all that's needed is adding `Public ` in front of the first token. Seems overkill and potentially irritating, I merged this PR because it fixes the tests, but clearly at least one test is missing for this inspection/fix, since it does not behave at it should - it's way too invasive for what it's doing.